specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/applicationGateway.json

```diff
@@ -451,6 +451,96 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableServerVariables": {
+      "get": {
+        "tags": [
+          "ApplicationGateways"
+        ],
+        "operationId": "ApplicationGateways_ListAvailableServerVariables",
+        "x-ms-examples": {
+          "Get Available Server Variables": {
+            "$ref": "./examples/ApplicationGatewayAvailableServerVariablesGet.json"
+          }
+        },
+        "description": "Lists all available server variables.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of all available server variables.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationGatewayAvailableServerVariablesResult"
+            }
+          }
+        }
+      }
+    },
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableRequestHeaders": {
+     "get": {
+        "tags": [
+          "ApplicationGateways"
+        ],
+        "operationId": "ApplicationGateways_ListAvailableRequestHeaders",
+        "x-ms-examples": {
+          "Get Available Request Headers": {
+            "$ref": "./examples/ApplicationGatewayAvailableRequestHeadersGet.json"
+          }
+        },
+        "description": "Lists all available request headers.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of all available request headers.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationGatewayAvailableRequestHeadersResult"
+            }
```

Comment:
Same as prior, please add a "default" error response

Reply:
We have only one response for this API. So it is by default "default".

Reply:
see comment above

Reply:
Added the "default" error response